app/src/main/java/com/qudus/tudee/designSystem/component/SnackBar.kt

```diff
@@ -0,0 +1,140 @@
+package com.qudus.tudee.designSystem.component
+
+import android.graphics.BlurMaskFilter
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.graphics.Paint
+import androidx.compose.ui.graphics.drawscope.drawIntoCanvas
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.graphics.toArgb
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.qudus.tudee.R
+import com.qudus.tudee.designSystem.theme.Theme
+import com.qudus.tudee.designSystem.theme.TudeeTheme
+
+
+@Composable
+fun SnackBar(
+    painter: Painter,
```

Comment:
We have only two states for error, so you can create a state and pass it instead of the Painter.

```kotlin
    Image(
                painter = when(state){
                // code
                },
                contentDescription = "snack bar",
                alignment = Alignment.Center,
                colorFilter = ColorFilter.tint(iconColor)
            )
```